### PR TITLE
Issue #318 - support datetime-tz

### PR DESCRIPTION
**Pull request by UltimateDogg**

### CODE REVIEW

**Review by ppodgorsek — CHANGES_REQUESTED** [review]
The PR looks good, I would however suggest adding automated tests and an explanation in the README about how it works.

**Comment by ppodgorsek** [comment]
Would it be possible to define the timezone as an environment variable? (by default, UTC) When running it, users could simply pass a new variable value instead of any complex setup.
This would probably require additional changes in the launch script.

**Comment by UltimateDogg** [comment]
> 
> 
> Would it be possible to define the timezone as an environment variable? (by default, UTC) When running it, users could simply pass a new variable value instead of any complex setup.
> This would probably require additional changes in the launch script.

yes timezones are everywhere, and i am not familiar with everything wed have to change. But this library at least makes it so you can account for differences between local time (UTC in this image) and the website you are on.

File: README.md [hunk 1]
```diff
@@ -92,6 +93,16 @@ This project includes the IMAP library which allows Robot Framework to connect t
 
 A suggestion to automate email testing is to run a [Mailcatcher instance in Docker which allows IMAP connections](https://github.com/estelora/docker-mailcatcher-imap). This will ensure emails are discarded once the tests have been run.
 
+### Dealing with Datetimes and Timezones
+
+This project is meant to allow your tests to run anywhere. Sometimes that can be in a different timezone than your local or the location under test. To help with that this library includes the [DateTimeTZ Library](https://testautomation.github.io/DateTimeTZ/doc/DateTimeTZ.html)
+
+To set the timezone used inside the docker image, you can set env variable TZ
```
Comment (ppodgorsek) [inline README.md:100]:
```suggestion
To set the timezone used inside the Docker image, you can set the `TZ` environment variable:
```

File: README.md [hunk 2]
```diff
@@ -92,6 +93,16 @@ This project includes the IMAP library which allows Robot Framework to connect t
 
 A suggestion to automate email testing is to run a [Mailcatcher instance in Docker which allows IMAP connections](https://github.com/estelora/docker-mailcatcher-imap). This will ensure emails are discarded once the tests have been run.
 
+### Dealing with Datetimes and Timezones
+
+This project is meant to allow your tests to run anywhere. Sometimes that can be in a different timezone than your local or the location under test. To help with that this library includes the [DateTimeTZ Library](https://testautomation.github.io/DateTimeTZ/doc/DateTimeTZ.html)
```
Comment (ppodgorsek) [inline README.md:98]:
```suggestion
This project is meant to allow your tests to run anywhere. Sometimes that can be in a different timezone than your local one or of the location under test. To help solve such issues, this image includes the [DateTimeTZ Library](https://testautomation.github.io/DateTimeTZ/doc/DateTimeTZ.html).
```

**Review by ppodgorsek — CHANGES_REQUESTED** [review]

**Review by ppodgorsek — APPROVED** [review]

**Pull request merged**